utils/build-mlir-air-pcie.sh

```diff
@@ -19,47 +19,51 @@
 # e.g. build-mlir-air-pcie.sh /scratch/llvm /scratch/mlir-aie
 #          /scratch/cmakeModules/cmakeModulesXilinx
 #
-# <cmakeModules dir> - cmakeModules, absolute directory
-# <mlir-aie dir>     - mlir-aie, absolute directory
+# <llvm dir>         - llvm
+# <cmakeModules dir> - cmakeModules
+# <mlir-aie dir>     - mlir-aie
 #
-# <mlir-air dir> - optional, mlir-air repo name, default is 'mlir-air'
-# <build dir>    - optional, mlir-air/build dir name, default is 'build-pcie'
-# <install dir>  - optional, mlir-air/install dir name, default is 'install-pcie'
+# <libXAIE dir>      - libXAIE to build runtime
+#
+# <build dir>    - optional, build dir name, default is 'build-pcie'
+# <install dir>  - optional, install dir name, default is 'install-pcie'
 #
 ##===----------------------------------------------------------------------===##
 
 if [ "$#" -lt 3 ]; then
     echo "ERROR: Needs at least 3 arguments for <llvm dir>, <cmakeModules dir> and <mlir-aie dir>."
     exit 1
 fi
-LLVM_DIR=$1
-CMAKEMODULES_DIR=$2
-MLIR_AIE_DIR=$3
+LLVM_DIR=`realpath $1`
+CMAKEMODULES_DIR=`realpath $2`
+MLIR_AIE_DIR=`realpath $3`
+
+LibXAIE_DIR=`realpath $4`
 
-MLIR_AIR_DIR=${4:-"mlir-air"}
 BUILD_DIR=${5:-"build-pcie"}
 INSTALL_DIR=${6:-"install-pcie"}
 
-mkdir -p $MLIR_AIR_DIR/$BUILD_DIR
-mkdir -p $MLIR_AIR_DIR/$INSTALL_DIR
-cd $MLIR_AIR_DIR/$BUILD_DIR
+mkdir -p $BUILD_DIR
+mkdir -p $INSTALL_DIR
+cd $BUILD_DIR
 
 PYTHON_ROOT=`pip3 show pybind11 | grep Location | awk '{print $2}'`
 
 cmake .. \
 	-GNinja \
 	-DCMAKE_INSTALL_PREFIX="../${INSTALL_DIR}" \
+	-DCMAKE_MODULE_PATH=${CMAKEMODULES_DIR}/ \
 	-DCMAKE_TOOLCHAIN_FILE=../cmake/modules/toolchain_x86.cmake \
 	-DLLVM_DIR=${LLVM_DIR}/build/lib/cmake/llvm \
 	-DMLIR_DIR=${LLVM_DIR}/build/lib/cmake/mlir \
 	-DAIE_DIR=${MLIR_AIE_DIR}/build/lib/cmake/aie \
 	-Dpybind11_DIR=${PYTHON_ROOT}/pybind11/share/cmake/pybind11 \
 	-DVitisSysroot="" \
+  -DLibXAIE_DIR=${LibXAIE_DIR} \
```

Comment:
Maybe clean up the rest to use spaces (not tabs) while we're at it? Also applies to build-mlir-air.sh and build-llvm-local.sh.